src/shared/Notifications/Notifications.scss

```diff
@@ -1,12 +1,19 @@
 @import '../../scss/global_styles';
 
+
+
 .notifications {
+  &__wrapper {
+    position: fixed;
+    top: 95px;
+    right: 20px;
+    z-index: 999;
+  }
+
+  display: block;
   color: $white;
```

Comment:
Please take a look at the css properties positioning best practices: https://9elements.com/css-rule-order/ . The color property should always be the last one, that's because of the way how browser rendering works: first it sets elements' sizes, positions, margins etc, and applying colors to the elements is the last process, so it is logical to place the color property in the end of the list.

Reply:
done)